ci-scripts/schedule_durable_tests.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+id=$(uuidgen)
+job_id=$(uuidgen)
+now=$(date +%s%N)
+ttl=$(date +%s -d "14 days")
+tag=${BUILD_TAG-latest}
+network=${1-dev}
+
+curl -s https://raw.githubusercontent.com/3box/pipeline-tools/develop/ci/scripts/schedule_job.sh | bash -s -- \
```

Comment:
Why does this script live in a different repo? Can it live here?

Reply:
It's a generic script to create a job in the CD manager DB. I was planning to invoke this from other repos that need to create CD manager jobs (e.g. deployment jobs), and have the CD manager repo be the place where all generic CI-related scripts are stored.

However, I don't have a strong opinion. I can duplicate that script wherever needed.

Reply:
Ah yes, if you run it with this URL, it'll work. It's in the CD manager PR and until it merges, it won't be in `develop`. I suppose I could just keep pointing to the branch from here then update this repo to point to CD manager `develop` after the other PR merges?
```
https://raw.githubusercontent.com/3box/pipeline-tools/feat/workflows/ci/scripts/schedule_job.sh
```

Reply:
I say duplicate it. Its a small script that basically just calls into docker which does the heavy lifting. Its hard to know who the dependencies of a script are if its just curled and executed. If we really want to share the logic I say we publish releases of a CLI tool. That way its clear there are downstream consumers of the tool, and it can be versioned and managed accordingly. 

However for such a small dependency that seems like too much work. Hence my recommendation to just duplicate it.

Reply:
Cool, done.

Also, I was able to hack it during testing but this won't run locally yet until the PR first goes in :(